testgen.sh

```diff
@@ -6,7 +6,7 @@ if [ $ENTRYPOINT ]; then
   entrypoint="$ENTRYPOINT"
 fi
 
-gens="react react-native next vue vuetify nuxt quasar typescript"
+gens="angular"
```

Comment:
Please keep tests for other generators, just add yours at the end of the list:

```suggestion
gens="react react-native next vue vuetify nuxt quasar typescript angular"
```

Reply:
It's just for development time. This will be rectified at final delivery